exercises/practice/allergies/.meta/tests.toml

```diff
@@ -1,189 +1,28 @@
-[canonical-tests]
+# This is an auto-generated file. Regular comments will be removed when this
+# file is regenerated. Regenerating will not touch any manually added keys,
+# so comments can be added in a "comment" key.
 
-# not allergic to anything
-# this test may not be in the suite but has not been intentionally rejected
-# "17fc7296-2440-4ac4-ad7b-d07c321bc5a0" = false
+[90c8f484-456b-41c4-82ba-2d08d93231c6]
+description = "allergic to everything"
 
-# allergic only to eggs
-# this test may not be in the suite but has not been intentionally rejected
-# "07ced27b-1da5-4c2e-8ae2-cb2791437546" = false
+[d2d71fd8-63d5-40f9-a627-fbdaf88caeab]
+description = "allergic to everything"
 
-# allergic to eggs and something else
-# this test may not be in the suite but has not been intentionally rejected
-# "5035b954-b6fa-4b9b-a487-dae69d8c5f96" = false
+[cc789c19-2b5e-4c67-b146-625dc8cfa34e]
+description = "allergic to everything"
```

Comment:
Looks like the descriptions got a bit mangled here.

Reply:
This is due to the test cases being nested. There is an issue for this: https://github.com/exercism/configlet/issues/202 We're working on a fix: https://github.com/exercism/configlet/pull/301

Reply:
Ok, cool.